corehq/apps/sms/templates/sms/usage_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load timezone_tags %}
+{% requirejs_main "sms/js/sms_usage_report" %}
+{% block page_content %}
+    <form method="post">
+        {% csrf_token %}
+        <div class="form-group">
+            <label for="date_range_selector">Select Date Range</label>
+            <input name="date_range" class="form-control" id="date_range_selector" data-startDate="" data-endDate="" placeholder="Date">
```

Comment:
Naming conventions: data identifiers should be hyphenated, so `data-startDate` becomes `data-start-date` (you can still refer to it as `startDate` in javascript).

Reply:
I figured out they were not actually required, I misunderstood the API in the first go.
But have moved the form to Form classes based on Manish's recommendation so the HTML of the form has been removed.